src/main/scala/gestalt/Quote.scala

```diff
@@ -171,31 +195,48 @@ abstract class Quote(val t: Toolbox, val toolboxName: String) {
           lift(pat)
       }
 
+      // FIXME does not match to any of ${name}.apply signatures
+      // For *Decl does not have a last argument,
+      //  tpe: expected: TypeTree, actual: Option[TypeTree],
+      // For *Def argument rhs: expected: Tree, actual: Option[Nothing]
       selectToolbox(name).appliedTo(mods, left, tpe, scalaNone)
```

Comment:
Feel free to create issues for `FIXME` if there are any discussion points regarding the fix. And feel free to create an issue and assign to me if it is a blocker.

FYI, to make the API simple, in a refactoring I reduced option types to a minimum by creating duplicate constructors.